distribution/pom.xml

```diff
@@ -171,7 +171,6 @@
 
     <modules>
         <module>fully-loaded</module>
```

Comment:
Ok. Thanks to @jpountz I finally found my error. So optional dependencies are really optional !

I was creating a shaded client example but I forgot to add `META-INF` Lucene dir with this:

```xml
<transformers>
    <transformer implementation="org.apache.maven.plugins.shade.resource.ManifestResourceTransformer" />
</transformers>
```

I have now a `qa/smoke-test-client` module running (PR will be sent soon). I can confirm that not other lib is needed to create a simple Transport Client...